explainer.md

```diff
@@ -113,41 +121,60 @@ The CSS images and computed properties/styles cached from Page-A are represented
 
 ```
 transition element
-└─ image wrapper
-   └─ image
+└─ states
+   └─ state
```

Comment:
I've added a new wrapper 😬, but it seems necessary to handle nesting. I guess we could omit this wrapper for transition elements that aren't a 'transition container', but is it better to always have it for consistency?

Reply:
My comment above touches on this DOM structure. Let's discuss it there? I don't think we should need to a new wrapper type but I might be missing the case you had in mind.